types/accedo__accedo-one/index.d.ts

```diff
@@ -6,7 +6,7 @@
 
 /**
  * @example
- * ~ import accedoOne from "@accedo/accedo-one"
+ * ~ import accedoOne = require("@accedo/accedo-one");
```

Comment:
You could leave this as-is since this documentation will be visible both to JS and to TS users, and `import...require` is TypeScript syntax.

| | JavaScript | TypeScript |
|-|-----------|-------------|
| CJS | `module.exports =` <br> `const x = require('./')` | `export =` <br> `import x = require('./')` |
| ES | `export default x` <br> `export { x }` <br> `import x from './'` <br> `import { x } from './'` | (Same as JS)

Reply:
Oh my... I think I have headache for all this. 😂
Ops, I inserted both in the tests I made but I forgot to commit it.
So, if you can use both in Typescript, what about inserting both with a comment? One for ES Syntax and the other for CJS.

Reply:
P.s. actually, I don't know if this is a VSCode problem, I see the whole comment only if I mouse over on the function in `.d.ts` but not in the code. I don't know if I'm doing something wrong.